code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -759,6 +759,18 @@
 	to_chat(target, span_userdanger("You're no longer identified as a nuclear operative! You are free to follow any valid goals you wish, even continuing to secure the disk. Just make sure neither any turrets nor operatives kill you on sight."))
 	return TRUE
 
+#define POSTERS_PROVIDED 3
+
+/obj/item/storage/box/syndie_kit/poster_box
+	name = "syndicate poster pack"
+	desc = "Contains a variety of demotivational posters to ensure minimum productivity for the crew of any Nanotrasen station."
+
+/obj/item/storage/box/syndie_kit/poster_box/PopulateContents()
+	for(var/i in 1 to POSTERS_PROVIDED)
+		new /obj/item/poster/traitor(src)
+
+#undef POSTERS_PROVIDED
```

Comment:
just make this a variable, you make your life easier

Reply:
Done!